assets/style.css

```diff
@@ -64,15 +64,33 @@ img {
   max-width: 100%;
 }
 
-*::-webkit-scrollbar {
-  width: 8px;
-  height: 8px;
-}
-*::-webkit-scrollbar-track {
-  background: none;
+@supports not selector(::-webkit-scrollbar) {
```

Comment:
Isn't it best to use `@supports (scrollbar-width: auto)`? 

Like in that example: https://developer.chrome.com/docs/css-ui/scrollbar-styling#supporting_older_browser_versions

Or would that not overwrite the styles correctly.

Reply:
Yep, that won't correctly override the styles.
If both are present, `scrollbar-*` will override all the `--webkit-scrollbar-*` properties.

This is due to the `CSS Scrollbars Styling Module Level 1` currently does not support the `scrollbar-button`.
https://drafts.csswg.org/css-scrollbars/#buttons